components/admin_console/license_settings/renew_license_link_card/renew_license_link_card.tsx

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+import {FormattedMessage} from 'react-intl';
+
+import moment from 'moment';
+
+import RenewalLink from 'components/announcement_bar/renewal_link/';
+
+import alertIcon from 'images/icons/round-blue-info-icon.svg';
+import warningIcon from 'images/icons/warning-red-icon.svg';
+
+import FormattedMarkdownMessage from 'components/formatted_markdown_message';
+
+import './renew_license_link_card.scss';
+
+export interface RenewLicenseLinkCardProps {
+    license: any;
+    isLicenseExpired: boolean;
+    totalUsers: number;
+}
+
+const RenewLicenseLinkCard: React.FC<RenewLicenseLinkCardProps> = ({license, totalUsers, isLicenseExpired}: RenewLicenseLinkCardProps) => {
```

Comment:
Possible name to simplify here, `Link` may not be adding any value here

```suggestion
const RenewLicenseCard: React.FC<RenewLicenseCardProps> = ({license, totalUsers, isLicenseExpired}: RenewLicenseCardProps) => {
```